css/index.css

```diff
@@ -1,3 +1,316 @@
 /* Use your own code or past solution for Great Idea Web Page CSS here! */
 
 
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+
+html, body, div, span, applet, object, iframe,
+h1, h2, h3, h4, h5, h6, p, blockquote, pre,
+a, abbr, acronym, address, big, cite, code,
+del, dfn, em, img, ins, kbd, q, s, samp,
+small, strike, strong, sub, sup, tt, var,
+b, u, i, center,
+dl, dt, dd, ol, ul, li,
+fieldset, form, label, legend,
+table, caption, tbody, tfoot, thead, tr, th, td,
+article, aside, canvas, details, embed, 
+figure, figcaption, footer, header, hgroup, 
+menu, nav, output, ruby, section, summary,
+time, mark, audio, video {
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Your code starts here! */
+
+
+
+.menu {
+	color:lightgray;
+	font-size: 15px;
+  font-family:'Titillium Web';
+  text-decoration:none;
+  margin-top:60px;
+  margin-left:80px;
+  display:inline-block;
+
+ 
+}
+
+.logo {
+
+  margin-left:370px;
+	margin-top:60px;
+	
+}
+
+h1  {
+  font-size:100px;
+  text-align:center;
+  margin-top:120px;
+  margin-left:300px;
+  display:inline-block;
+  
+}
+
+.pic{
+  display:inline-block;
+  height:400px;
+  width:400px;
+  border-radius:100%;
+ position:absolute;
+ margin-top:70px;
+ margin-left:20px;
+}
+
+.start {
+	border:1px solid black;
+  display:inline-block;
+  padding:10px 60px;
+  font-family:'Titillium Web';
+  position:absolute;
```

Comment:
I think all of these position absolutes are creating some issues for you. Try and use those sparingly.